src/arrayMethodSplice.js

```diff
@@ -5,7 +5,56 @@
  */
 function applyCustomSplice() {
   [].__proto__.splice2 = function(start, deleteCount, ...items) {
-    // write code here
+    const leftArr = [];
+    const rightArr = [];
+    const removedElements = [];
+
+    if (arguments.length < 1 || start > this.length || deleteCount < 0) {
+      return [];
+    }
+
+    let removedPoint = 0;
```

Comment:
seems like you removed some point and created a variable for this point,
this is a better option `startPoint`